api/v1/sriovnetworknodestate_types.go

```diff
@@ -119,6 +119,9 @@ type System struct {
 	// +kubebuilder:validation:Enum=shared;exclusive
 	//RDMA subsystem. Allowed value "shared", "exclusive".
 	RdmaMode string `json:"rdmaMode,omitempty"`
+	// OVS config. It will be provided for ovs-vswitchd service as other_config option
+	// +kubebuilder:default:={hw-offload: "true"}
```

Comment:
Do we really want such a default at the API level?
If I understand correctly, this means that if a user sets ovsConfig = `{"foo":"bar"}`, then the default will not apply.
This may be confusing. Should we instead always add `{hw-offload: "true"}` to the ovsConfig value when it doesn't contain an explicit value for this field?